openlibrary/templates/site/banner.html

```diff
@@ -0,0 +1,14 @@
+$def with (announcement, storage_key)
+
+$# Takes following parameters
+$# * announcement (str) -- Rendered content that will be displayed in the banner
+$# * storage_key (str) -- identifier for the banner. If the user dismisses the banner, storage_key will be added to the user's preferences to prevent the banner from showing again.
```

Comment:
```suggestion
$# Renders a dismissable banner element containing the given announcement
$# announcement (str) -- Rendered content that will be displayed in the banner
$# storage_key (str) -- identifier for the banner. If the user dismisses the banner, storage_key will be added to the user's preferences to prevent the banner from showing again.
```